src/commands/chatInput/data/reset/scoreboard.ts

```diff
@@ -0,0 +1,59 @@
+import { ButtonStyle, ComponentType } from "discord.js";
+import type { ChatInputCommand } from "../..";
+import { components } from "../../../../handlers/interactions/components";
+import { databaseLogger } from "../../../../utils/logger/database";
+
+const command: ChatInputCommand = {
```

Comment:
user should be required to have selected a counting channel

Reply:
fixed